Assets/BossRoom/Scripts/Server/Game/Character/ServerCharacterMovement.cs

```diff
@@ -69,6 +73,13 @@ public override void OnNetworkSpawn()
         /// <param name="position">Position in world space to path to. </param>
         public void SetMovementTarget(Vector3 position)
         {
+#if UNITY_EDITOR || DEVELOPMENT_BUILD
+            if (TeleportModeActivated)
+            {
+                Teleport(position);
```

Comment:
Should this use NetworkTransform's teleport? (which disables interpolation for that movement)

Reply:
good point, I'll look into it

Reply:
Although, this is the same Teleport used by the dash action, and we already have a ticket to refactor this (https://jira.unity3d.com/browse/MTT-1743) so it should probably be part of another PR